scrapers/PinkVisual.yml

```
@@ -0,0 +1,32 @@
+name: pinkvisual
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.ipinkvisualpass.com/es/sx/pc/episode
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //div[@class="summary"]/h3[@itemprop="name"]/text()
+      Details: //span[@itemprop="description"]/text()
```

This matches details from the `other scenes` as well
The below gets a single match
```suggestion
      Details: //p[@class="description"]/span[@itemprop="description"]/text()
```